### PR TITLE
Consider new abstract role "phrase"

### LINKED ISSUE #908: Consider new abstract role "phrase"
Consider an abstract role for the ontology which would be used for roles equivalent to HTML phrasing elements. Section doesn't seem correct as a parent role for many of these and there doesn't seem to be any other place to put them... Example would be the new subcript/superscript roles

### PR DESCRIPTION
**Pull request by jnurthen**
Fixes #908

Have not spent much time on the actual definition and the name.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1814.html" title="Last updated on Sep 29, 2022, 12:51 AM UTC (4ee6403)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1814/bcf44d1...4ee6403.html" title="Last updated on Sep 29, 2022, 12:51 AM UTC (4ee6403)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3610,7 +3610,7 @@ <h2>Definition of Roles</h2>
 					</tr>
 					<tr>
 						<th class="role-parent-head" scope="row">Superclass Role:</th>
-						<td class="role-parent"><rref>structure</rref></td>
+						<td class="role-parent"><rref>phrasingcontent</rref></td>
```
Comment (pkra) [inline index.html:3613]: I find this one confusing. The generic role specifically mentions `div` which does not match well.

File: index.html [hunk 2]
```diff
@@ -1795,7 +1795,7 @@ <h2>Definition of Roles</h2>
 					</tr>
 					<tr>
 						<th class="role-parent-head" scope="row">Superclass Role:</th>
-						<td class="role-parent"><rref>section</rref></td>
+						<td class="role-parent"><rref>phrasingcontent</rref></td>
```
Comment (pkra) [inline index.html:1798]: This feels strange to me; captions tend to allow flow content, e.g., table or figure captions. (Also, all the examples for the caption role use `div`.)

File: index.html (hunk 1 above)
Comment (jnurthen) [reply index.html:3613]: yet span does ;) - no idea for a solution though!

**Review by cookiecrook — APPROVED** [review]
FWIW I found this StackOverflow helpful in sorting out the diffs between HTML flow content and phrasing content. 
https://stackoverflow.com/questions/30233447/what-is-the-difference-between-phrasing-content-and-flow-content

I dislike the role name, but since that's what HTML calls it, I think it's fine as is. Thanks @jnurthen

**Comment by pkra** [comment]
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories is also a good reference.

**Comment by jnurthen** [comment]
Looking at the roles we are discussing - this is very different from HTML Phrasing Content which includes all sorts of interactives. Using the same term would IMO cause confusion.
I've been trying to find a good term but haven't yet come up with one. Until we do I propose that we park this as changing abstract roles from one to another doesn't sound useful unless we can get the naming right.

**Comment by scottaohara** [comment]
guess `textyMcTextFace` just didn't cut it...

**Pull request closed without merging**